tests/test_pyproject_toml.py

```diff
@@ -40,7 +40,7 @@ def test_pyproject_toml_works(capsys: "CaptureFixture") -> None:
     assert out == expected_out
 
 
-def test_cli_overwrites_pyprojecttoml(capsys: "CaptureFixture") -> None:
+def test_cli_extends_pyprojecttoml(capsys: "CaptureFixture") -> None:
```

Comment:
Should this existing test be modified like this or should I make a new test and leave the old one (it still passes, but the semantics of the name might be confusing)?

Also is this an appropriate location for the test?

Reply:
Yeah, if the old test's name/behaviour no longer make sense, I think it's OK to overwrite

Reply:
I was thinking again about the old test that checked CLI arguments overrode `pyproject.toml` configuration. That is probably still behaviour that should be checked for arguments to nbQA rather than the chosen tool.

E.g. perhaps there should be a test that applies e.g. `--nbqa-skip-cell-tags` at command line and also in `pyproject.toml` and verifies that CLI takes precedence?

Let me know if you want to include something like that as part of this PR or whether it can wait for follow-up work.